build.gradle

```diff
@@ -197,6 +188,8 @@ configure(javaProjects) { subproject ->
     apply plugin: 'kotlin'
     apply plugin: 'kotlin-spring'
     apply plugin: 'io.spring.nullability'
+    apply plugin: 'org.jetbrains.dokka'
```

Comment:
This is not correct: it is waste of resource to apply the plugin in the sub-project where is nothing with Kotlin.